wsidicom/instance.py

```diff
@@ -1082,13 +1285,115 @@ def get_encoded_tiles(
         z: float,
         path: str
     ) -> List[bytes]:
-        """Return image bytes for tile defined by tile (x, y), z, and optical
-        path."""
+        """Return tiles for tile defined by tile (x, y), z, and optical
+        path.
+
+        Parameters
+        ----------
+        tiles: List[Point]
+            Tiles to get.
+        z: float
+            Z coordinate.
+        path: str
+            Optical path.
+
+        Returns
+        ----------
+        List[bytes]
+            Tiles in bytes.
+        """
         return [
             self._get_encoded_tile(tile, z, path) for tile in tiles
         ]
 
-    def get_encoded_scaled_tiles(
+    def get_scaled_tile(
```

Comment:
Now when people have started to use the library, renaming public functions is a bigger deal as you may break other peoples software. We have not promised a frozen API at this stage, but maybe we should document all renames in the release notes?

Reply:
Noted